WebApp/WebContent/resources/style/main.css

```diff
@@ -692,3 +692,13 @@ table.fileDates {
    75% {background-color: #fd0;}
   100% {background-color: #fff;}
 }
+
+#applicationMenu > ul > li:last-child {
```

Comment:
Is it possible to put a class name on the tab, instead of relying on `last-child`? There's going to be another menu entry to put on the right in the future.